cmd/influx/pkg_test.go

```diff
@@ -126,6 +131,24 @@ func Test_Pkg(t *testing.T) {
 					Version:     "new version",
 				},
 			},
+			{
+				pkgFileArgs: pkgFileArgs{
+					name:     "yaml out",
+					encoding: pkger.EncodingYAML,
+					filename: "pkg_0.yml",
+					flags: []flagArg{
+						{name: "name", val: "new name"},
+						{name: "description", val: "new desc"},
+						{name: "version", val: "new version"},
+						{name: "org", val: "influxdata"},
+					},
+				},
+				expectedMeta: pkger.Metadata{
```

Comment:
we need a test for each cmd that updated with org stuff. You'll need to add this to export(export all is separate) func

Reply:
not 100% sure I follow ya

Reply:
the export cmd and export all cmd were obth updated. but only 1 test was added. There should be an additional test for the export cmd. You added to the export all. BUt the other is missing

Reply:
yeah hm. does exporting resources by organization id exist?

Reply:
the org id is needed for the exporting of resources, any resource, since the `orgID` is necessary for many of the resource lookups. The export all exports everything for an org, which you have covered in the test right now, so we're good there. The tests are only testing that the correct params are being passed/accessed really. Just tests our public API for the CI

Reply:
I understand, but where I'm getting at is the implementation of `influx pkg export` doesn't really accept an organization id as input. If that holds true, I'm going to remove the flag for now from the export.

Reply:
ahhh yah, your'e right man, that got ripped out a while back 🤦‍♂